reproject/mosaicking/coadd.py

```diff
@@ -211,7 +211,12 @@ def reproject_and_coadd(
             if input_weights is None:
                 weights_in = None
             else:
-                weights_in = parse_input_weights(input_weights[idata], hdu_weights=hdu_weights)
+                weights_in, weights_wcs = parse_input_weights(
+                    input_weights[idata], hdu_weights=hdu_weights, return_wcs=True
```

Comment:
I think this is the only place parse_input_weights is used so I don't think we need the kwarg, we can just always return the WCS

Reply:
fine by me, i can remove the kwarg